src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -1720,14 +1729,18 @@ private bool EatTid(TokKind tid)
 
         // Returns the current token if it's of the given kind and moves to the next token.
         // If the token is not the right kind, reports an error, leaves the token, and returns null.

```

Comment:
>reports an error [](http://example.com/codeflow?start=47&length=16)

Please update the comment to account for the new argument. May be a good time to convert this comment to the `///` format.